GiniVisionExample/GiniVisionExample/Screens/Results/ResultsViewController.xib

```diff
@@ -28,7 +28,7 @@
                 <view contentMode="scaleToFill" translatesAutoresizingMaskIntoConstraints="NO" id="nou-8a-j5B">
                     <rect key="frame" x="0.0" y="20" width="375" height="51.5"/>
                     <subviews>
-                        <label opaque="NO" userInteractionEnabled="NO" contentMode="left" horizontalHuggingPriority="251" verticalHuggingPriority="251" text="Ergebnisse die für Zahlungsüberweisungen wichtig sind befinden sich am Anfang der Liste. Bitte überprüfen Sie die Ergebnisse." textAlignment="natural" lineBreakMode="tailTruncation" numberOfLines="0" baselineAdjustment="alignBaselines" adjustsFontSizeToFit="NO" translatesAutoresizingMaskIntoConstraints="NO" id="U6A-kf-wmn">
+                        <label opaque="NO" userInteractionEnabled="NO" contentMode="left" horizontalHuggingPriority="251" verticalHuggingPriority="251" text="Ergebnisse, die für Zahlungsüberweisungen wichtig sind, befinden sich am Anfang der Liste. Bitte überprüfen Sie die Ergebnisse." textAlignment="natural" lineBreakMode="tailTruncation" numberOfLines="0" baselineAdjustment="alignBaselines" adjustsFontSizeToFit="NO" translatesAutoresizingMaskIntoConstraints="NO" id="U6A-kf-wmn">
```

Comment:
One issue, one PR!!!

Reply:
I know but I did not want to bother you with several PR.

Reply:
It's not a problem.